uk.po

```diff
@@ -601,7 +601,7 @@ msgstr "корейська"
 
 #: dialect/languages.py:65
 msgid "Kurdish (Kurmanji)"
-msgstr "курдська (Курманджі)"
+msgstr "курдська"
```

Comment:
I wonder if this is an important distinction to make? And if so, I wonder how many of these cases are there...

Reply:
Yes I also saw that and had that doubt, but also in other languages the CLDR names clarify languages like that if need, like in the case of the Chinese variants, but without the parentheses. So I guess it's a decision made by the CLDR translator.

Also let's remember that our English strings come from Google, so maybe we should start using the CLDR English names for each lang code instead of the current ones?

Reply:
> Also let's remember that our English strings come from Google, so maybe we should start using the CLDR English names for each lang code instead of the current ones?

Yeah, I think that would be the correct fix.